variable_buffer_rw.js

```diff
@@ -66,7 +69,8 @@ VariableBufferRW.prototype.writeInto = function writeInto(buf, buffer, offset) {
     return new WriteResult(null, start + length);
 };
 
-VariableBufferRW.prototype.readFrom = function readFrom(buffer, offset) {
+VariableBufferRW.prototype.readFrom =
```

Comment:
nitpick: for type stability you probably do not want to shadow this and have `readFrom` always be a field rather then on the prototype.